osm_fieldwork/CSVDump.py

```diff
@@ -32,7 +32,11 @@
 from osm_fieldwork.osmfile import OsmFile
 from osm_fieldwork.xlsforms import xlsforms_path
 
+#Instantiate logger
+log_level = os.getenv("LOG_LEVEL", default="INFO")
 # set log level for urlib
+logging.getLogger("urllib3").setLevel(log_level)
```

Comment:
Do we use urllib anywhere?

Reply:
I don't see urllib3 being used anywhere but I noticed it was set in certain modules so I maintained that.
It is still in the pdm.lock file and it was stated in the 0.1.0 changelog as used for verifying certs.

I can go ahead and remove those lines if its no longer in use.

Reply:
Good point! A package we use probably uses urllib underneath, possibly `requests`. 

We can keep this line to suppress the logs 👍